tests/unit_tests/getMatchingRow.test.js

```diff
@@ -42,3 +42,18 @@ it("matches the right table and field rows for yesnomaybe buttons", async functi
 });
 
 
+it("matches the right table and field rows for yesnomaybe buttons", async function() {
```

Comment:
I think this should say "... for other buttons"

Reply:
Thanks!